interface/core.xhr.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "core.xhr",
+  "api": {
+    // @param {String} method - GET/POST/DELETE
+    // @param {String} url
+    // @param {Boolean} async
```

Comment:
Idea is to mimic the exact parameters in the XHR spec.

On Wed, Jan 7, 2015 at 1:11 PM, Benjamin M. Schwartz <
notifications@github.com> wrote:

> In interface/core.xhr.json
> <https://github.com/freedomjs/freedom/pull/188#discussion-diff-22603223>:
>
> > @@ -0,0 +1,41 @@
> > +{
> > +  "name": "core.xhr",
> > +  "api": {
> > +    // @param {String} method - GET/POST/DELETE
> > +    // @param {String} url
> > +    // @param {Boolean} async
>
> Is there a good reason to include this? Given that the interface is always
> asynchronous, I'd be inclined to remove it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom/pull/188/files#r22603223>.
>